packages/synth.queues/src/services/queue.service.ts

```diff
@@ -7,7 +7,12 @@ export abstract class QueueService extends Service {
     public abstract receiveMessage(queueUniqueTag: string): Promise<Message>;
 
     /** Creates queue and returns `queueUniqueTag` */
-    public abstract createQueue(queueName: string): Promise<Queue>;
+    public abstract createQueue(
+        queueName: string,
+        numPartitions?: number,
+        replicationFactor?: number,
+        topicConfig?: object
+    ): Promise<Queue>;
```

Comment:
other message brokers do not contain such options. you should get numPartitions, replicationFactor, topicConfig from config